### PR TITLE
Add first snippet to create a basic tool template

### PR DESCRIPTION
**Pull request by davelopez**
Configured the VSCode extension to contribute snippets defined in [snippets.json](https://github.com/davelopez/galaxy-tools-extension/compare/snippets?expand=1#diff-663e38ec260b17239e542bd620f28f2e) file.

The ``gtool`` snippet creates the basic structure of a Galaxy Tool.
snippets

More snippets can be contributed to this file following the snippets syntax [here](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax)

### CODE REVIEW

File: client/src/snippets.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (bgruening) [inline client/src/snippets.json:1]: Cool, can you add a second example so we know how this json is structured. Maybe for something like:

```
                <conditional name="output_options">
                    <param name="reads_report_type" type="select" label="What would you like to have reported?">
                        <option value="retained">The actual reads</option>
                        <option value="count">The count of reads (-c)</option>
                    </param>
                    <when value="retained">
                    </when>
                    <when value="count" />
                </conditional>
```

File: client/src/snippets.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "Basic Galaxy tool": {
+        "prefix": "gtool",
```
Comment (bgruening) [inline client/src/snippets.json:3]: I see this is too easy :)
But I guess we should have a common prefix?

Would `gx-` work?

**Review by bgruening — APPROVED** [review]
Cool, this should be added to the CONTRIBUTION and/or Readme file. How to add more snippets :)

File: client/src/snippets.json (hunk 2 above)
Comment (davelopez) [reply client/src/snippets.json:3]: Yes ``gx-`` could be a good starting prefix for all snippets. I'll update the sample to ``gx-tool``

File: client/src/snippets.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            "        <option value=\"$5\">TODO: option $5</option>",
+            "    </param>",
+            "    <when value=\"$4\">",
+            "        $0",
+            "    </when>",
+            "    <when value=\"$5\">",
+            "    </when>",
+            "</conditional>"
+        ],
+        "description": "Conditional parameter select with two options"
```
Comment (davelopez) [inline client/src/snippets.json:38]: This can be a sample of how to include a second snippet.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Very nice!

**Pull request merged**